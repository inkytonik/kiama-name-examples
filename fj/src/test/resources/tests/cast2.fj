class A extends Object {
  A() {
    super();
  }
}
(A) (Object) new A()
