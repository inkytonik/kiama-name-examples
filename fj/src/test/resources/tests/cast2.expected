src/test/resources/tests/cast2.fj:6:1:info: (A)(Object ...: A
(A) (Object) new A()
^
src/test/resources/tests/cast2.fj:6:5:info: (Object)ne ...: Object
(A) (Object) new A()
    ^
src/test/resources/tests/cast2.fj:6:14:info: new A(): A
(A) (Object) new A()
             ^
A
